Packs/Recorded_Future/Integrations/Recorded_Future/Recorded_Future.py

```diff
@@ -58,7 +58,7 @@ def http_request(method, url_suffix, params=None):
             else:
                 error_str = 'Request failed, status: ' + str(res.status_code) + ', details: ' + res.text
             return_error(error_str)
-    except Exception, e:
+    except Exception as e:
```

Comment:
This is python 2 integration, not python 3

Reply:
This syntax works in BOTH.